web_src/js/features/repo-issue.js

```diff
@@ -160,6 +160,19 @@ export function initRepoIssueCommentDelete() {
         _csrf: csrfToken,
       }).done(() => {
         const $conversationHolder = $this.closest('.conversation-holder');
+
+        // Check if this was a pending comment.
+        if ($conversationHolder.find('.pending-label').length) {
+          // This is a pending comment.
+          const commentCounter = document.querySelector('#review-box .review-comments-counter');
+          // Decrease the counter by one.
+          commentCounter.textContent = parseInt(commentCounter.textContent) - 1;
```

Comment:
```suggestion
          commentCounter.textContent = String(parseInt(commentCounter.textContent) - 1);
```

Interface only takes string, we should not rely on coercion.